src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random rand;
+
     public String getRandomColor() {
-        return null;
+        int i = rand.nextInt(Color.values().length);
+        return Color.values()[i].toString();
```

Comment:
```suggestion
        return Color.values()[i].name();
```
read checklist more carefully
<img width="995" alt="Screen Shot 2022-10-17 at 8 35 17 PM" src="https://user-images.githubusercontent.com/90692308/196244973-ce1523f4-ce69-4f5b-a549-221383ccf9e9.png">